docs/manuals/source/TasksAndConcepts/Plugins/FileDaemonPlugins/PythonFdPlugin.rst.inc

```diff
@@ -71,4 +71,4 @@ Example:
      }
    }
 
-This plugin bareos-fd-file-interact from https://github.com/bareos/bareos/tree/master/contrib/fd-plugins/options-plugin-sample has a method that is called before and after each file that goes into the backup, it can be used as a template for whatever plugin wants to interact with files before or after backup.
+This plugin bareos-fd-file-interact from https://github.com/bareos/bareos/tree/master/contrib/fd-plugins/bareos_option_example has a method that is called before and after each file that goes into the backup, it can be used as a template for whatever plugin wants to interact with files before or after backup.
```

Comment:
While this change is definitely better, there is more to do to use this example plugin. The `bareos_option_example` directory must be copied and renamed to `/usr/lib/bareos/plugins/bareos-fd-file-interact`.
Maybe you also want to explain this (or adapt this section and move it to the Developer Guide section).